manticore/platforms/linux.py

```diff
@@ -1412,7 +1412,11 @@ def sys_arch_prctl(self, code, addr):
         ARCH_SET_FS = 0x1002
         ARCH_GET_FS = 0x1003
         ARCH_GET_GS = 0x1004
-        assert code == ARCH_SET_FS
+        if code not in (ARCH_SET_GS, ARCH_SET_FS, ARCH_GET_FS, ARCH_GET_GS):
```

Comment:
```suggestion
        if code not in {ARCH_SET_GS, ARCH_SET_FS, ARCH_GET_FS, ARCH_GET_GS}:
```

It doesn't matter here but usually it is better to use set literals instead of a tuple/list. Sets have average of O(1) while lists have O(n) for `item in collection` ([source](https://wiki.python.org/moin/TimeComplexity)).

It can also be tested in ipython interactive shell:
![image](https://user-images.githubusercontent.com/10009354/50634859-30adaf00-0f50-11e9-9f77-d11bcbc2a9d9.png)